packages/remix/src/generators/application/application.impl.ts

```diff
@@ -34,6 +34,32 @@ export default async function (tree: Tree, _options: NxRemixGeneratorSchema) {
     sourceRoot: `${options.projectRoot}`,
     projectType: 'application',
     tags: options.parsedTags,
+    targets: {
+      build: {
+        command: `remix build`,
+        options: {
+          cwd: options.projectRoot,
+        },
+      },
+      serve: {
```

Comment:
There's a question around whether we want this to be `dev` to match the remix command or `serve` to keep consistency with Nx repos.

Reply:
I think `serve` is fine, we do the same with Next.js where `nx serve` is basically `next dev`. And also `@nx/cypress:cypress-project` assumes the `serve` target.